pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/definition/validation/DecimalRangeRuleDef.java

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.ddp.model.activity.definition.validation;
+
+import com.google.gson.annotations.SerializedName;
+import org.broadinstitute.ddp.model.activity.definition.template.Template;
+import org.broadinstitute.ddp.model.activity.definition.types.DecimalDef;
+import org.broadinstitute.ddp.model.activity.types.RuleType;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+public class DecimalRangeRuleDef extends RuleDef {
+
+    @SerializedName("min")
+    private DecimalDef min;
+
+    @SerializedName("max")
+    private DecimalDef max;
+
+    public DecimalRangeRuleDef(Template hintTemplate, DecimalDef min, DecimalDef max) {
+        super(RuleType.DECIMAL_RANGE, hintTemplate);
+        if (min != null && max != null && min.compareTo(max) > 0) {
+            throw new IllegalArgumentException("Must provide a valid range");
+        }
+        this.min = min;
+        this.max = max;
+    }
```

Comment:
Not seeing a DECIMAL_RANGE in the current ReDoc docs. @bskinner

Reply:
That's a known issue- @Trimax and I noticed it was missing when we were going over this PR.

Reply:
I'm not sure I got what you mean. Are you offering to remove the NumericRange and use DecimalRange everywhere since it's more general?